BookingsAPI/Bookings.IntegrationTests/Features/SuitabilityAnswer.feature

```diff
@@ -0,0 +1,11 @@
+﻿
+Feature: SuitabilityAnswer
+	In order to view all suitability answers
+	As an api service
+	I want to be able to retrieve suitability data
+@VIH-4460
+Scenario: Get all suitability answers sorted by latest one 
```

Comment:
Can we add a test for the cursor as well? To get the 2nd page, like if there are two suitability questionnaires, first get it with limit = 1 then get the 2nd page

Reply:
Ok